release-notes/opensearch-flow-framework.release-notes-2.13.0.0.md

```diff
@@ -0,0 +1,20 @@
+## 2024-03-18 Version 2.13.0.0
```

Comment:
I usually see this title in other release-notes: `## Version 2.13.0.0 Release Notes` is this also common to have date there?

Reply:
I took the same format as our 2.12 release notes

Reply:
Reference : https://github.com/opensearch-project/flow-framework/pull/467#discussion_r1471602226

Reply:
I guess core does this but never seen a plugin have that, if it passed in 2.12 from build time then I am cool with it